packages/nextjs/components/scaffold-eth/index.tsx

```diff
@@ -1,3 +1,4 @@
+"use client";
```

Comment:
don't do this please. You need to add 'use client'; explicitly for every component. Half of components here works fine as server components